src/main/java/org/minions/devfund/alejandro/WhackAMole.java

```diff
@@ -0,0 +1,182 @@
+package org.minions.devfund.alejandro;
+
+/**
+ * Class for whack a mole.
+ */
+public class WhackAMole {
+
+    private int score;
+    private int molesLeft;
+    private int moles;
+    private int attemptsLeft;
+    private final char[][] moleGrid;
+    private final int size;
+    private final char mole = 'M';
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Make this final field static too. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1170)